Source/WebCore/css/html.css

```diff
@@ -1124,7 +1123,6 @@ select {
 
 #if !(defined(WTF_PLATFORM_IOS_FAMILY) && WTF_PLATFORM_IOS_FAMILY)
 select:is([size], [multiple]), select[size][multiple] {
-    align-items: flex-start;
```

Comment:
Can you leave that unchanged for this bug? We should remove it, but also remove `align-items: center` to avoid getting incorrect results.

Reply:
Can do. :-)

Reply:
Done.